docs/json/sonarr/v4/anime/Anime 10bit.json

```
@@ -0,0 +1,26 @@
+{
+  "trash_id": "xxx",
+  "trash_score": "0",
+  "name": "Anime 10bit",
+  "includeCustomFormatWhenRenaming": false,
+  "specifications": [
+    {
+      "name": "10bit",
+      "implementation": "ReleaseTitleSpecification",
+      "negate": false,
+      "required": false,
+      "fields": {
+        "value": "10.?bit"

```

I would use this for optional `.`, and not any character (except for line terminators).
```suggestion
        "value": "10\\.?bit"
```